### PR TITLE
refactor: minor refactor in the code

### PR DESCRIPTION
**Pull request by ansabgillani**
This PR caters small refactoring in various locations within the codebase.

[PROD-2389](https://openedx.atlassian.net/browse/PROD-2389)

### CODE REVIEW

**Review by DawoudSheraz — APPROVED** [review]

**Review by Ali-D-Akbar — APPROVED** [review]

File: src/utils/index.test.js [hunk 1]
```diff
@@ -191,4 +192,13 @@ describe('Test Utils', () => {
       'lms_username',
     ]);
   });
+
+  describe('Extract Search Params', () => {
```
Comment (azanbinzahid) [inline src/utils/index.test.js:196]: Please add a test when input is empty or null.

**Review by azanbinzahid — APPROVED** [review]
Please

**Comment by DawoudSheraz** [comment]
@ansabgillani Please update the PR and merge it at the earliest. Thank you

File: src/utils/index.test.js [hunk 2]
```diff
@@ -191,4 +192,21 @@ describe('Test Utils', () => {
...
+      const params = extractParams(query);
+      expect(params.size).toEqual(3);
+      expect(params.get('param1')).toEqual('test');
+      expect(params.get('param2')).toEqual('test2');
+      expect(params.get('param3')).toEqual('');
+    });
+    it('no search param', () => {
+      const query = '';
+      const params = extractParams(query);
+      console.log(params);
```
Comment (DawoudSheraz) [inline src/utils/index.test.js]: nit: to remove this console statement
